Example/Example/MoreAttachmentRenderer.swift

```diff
@@ -30,7 +30,11 @@ extension MoreAttachmentRenderer: TextViewAttachmentImageProvider {
         UIGraphicsBeginImageContextWithOptions(size, false, 0)
 
         let label = attachment.text.uppercased()
-        let attributes = [NSForegroundColorAttributeName: textColor]
+
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.baseWritingDirection = .leftToRight
```

Comment:
@frosty humm, I'm just not sure is regarding if on an RTL language the .leftToRight forces the text to be written MORE instead of EROM...